json_spec/output/source_location.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/opencomputeproject/ocp-diag-core/sourceLocation",
+    "title": "Source Location",
+    "description": "TODO: link to md doc",
```

Comment:
comment above about filling these in applies everywhere

Reply:
updated